src/adagrams.js

```diff
@@ -1,6 +1,127 @@
 const Adagrams = {
+  poolOfLetters: [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A',
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D',
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E',
+    'F', 'F',
+    'G', 'G', 'G',
+    'H', 'H',
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I',
+    'J',
+    'K',
+    'L', 'L', 'L', 'L',
+    'M', 'M',
+    'N', 'N', 'N', 'N', 'N', 'N',
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O',
+    'P', 'P',
+    'Q',
+    'R', 'R', 'R', 'R', 'R', 'R',
+    'S', 'S', 'S', 'S',
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U',
+    'V', 'V',
+    'W', 'W',
+    'X',
+    'Y', 'Y',
+    'Z',
+  ],
+
+  getrandNum() {
+    return Math.floor(Math.random() * this.poolOfLetters.length)
+  },
+
   drawLetters() {
-    // Implement this method for wave 1
+    const userHand = [];
+
+    while (userHand.length < 10) {
+      let randNum = this.getrandNum();
+      let randLetter = this.poolOfLetters[randNum];
+      userHand.push(randLetter);
+      this.poolOfLetters.splice(randNum, 1);
```

Comment:
You need to make a copy of `poolOfLetters`.  

Your code as it's written doesn't put the letters back into the pool for future hands.

(This logic is tricky to get exactly right.  There's a task to improve the tests here for next cohort to catch issues like this.)